components/security-mgt/org.wso2.carbon.security.mgt.ui/src/main/resources/web/securityconfig/index.jsp

```diff
@@ -20,8 +20,8 @@
 <%@page import="org.apache.axis2.context.ConfigurationContext" %>
 <%@page
         import="org.wso2.carbon.CarbonConstants" %>
-<%@page import="org.wso2.carbon.security.mgt.stub.config.xsd.SecurityScenarioData" %>
```

Comment:
Not sure if this is the correct approach. This UI component should use the classes from the stub.

Reply:
It uses the classes from the stub which is represented with `xsd`just need to configure to build it as a stub. But right now seems there are some problem with the generation too will discuss this with you offline.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/2921/commits/c7d87531d9707a98ee5b6f19c9ff1f12770e5cd7.